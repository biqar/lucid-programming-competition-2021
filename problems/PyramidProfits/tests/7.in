1
657555 Jude 60
1000
657555 470000
657555 1417700
Jude 1831000
657555 1174000
Jude 362800
657555 285200
Jude 178400
Jude 50300
Jude 1313800
Jude 1970900
Jude 1977300
657555 1458600
657555 459800
Jude 787700
Jude 1752300
Jude 1108300
Jude 1374100
Jude 683800
Jude 915000
Jude 1912500
657555 1741000
657555 7200
Jude 1591600
Jude 739100
Jude 440000
Jude 996600
Jude 973700
Jude 1798500
657555 1295800
Jude 418000
657555 120900
657555 1344800
657555 1958000
Jude 1546200
657555 1394800
657555 1197500
Jude 411500
Jude 1215400
657555 1866000
657555 1818100
Jude 1157600
657555 202500
657555 562400
657555 821600
Jude 1813800
Jude 1201200
657555 1102500
Jude 706800
657555 1977900
Jude 681800
657555 651800
Jude 1457300
657555 155800
657555 592300
Jude 1653200
Jude 739000
Jude 1489800
Jude 1699900
657555 1684900
Jude 1808700
Jude 1401800
657555 994900
Jude 399400
657555 204800
657555 285900
Jude 453700
Jude 1411500
657555 1303500
Jude 494900
657555 1368600
Jude 408200
657555 834000
657555 287300
Jude 179900
657555 700500
657555 1496600
657555 1355800
Jude 114500
657555 836000
657555 686700
657555 1592300
657555 1974500
Jude 1637400
657555 852600
Jude 528500
657555 818500
657555 39900
Jude 886900
Jude 1197300
Jude 873300
657555 248300
657555 1264600
657555 975700
Jude 1615400
Jude 1007700
657555 200700
657555 876500
Jude 499900
Jude 1362300
657555 604700
Jude 1102900
Jude 479000
657555 216300
657555 1449800
Jude 1775000
657555 250300
657555 1623800
Jude 1789700
Jude 316500
657555 703300
657555 1536300
657555 680700
Jude 1098800
657555 931700
Jude 1420600
Jude 544600
657555 502400
657555 401200
657555 1148100
657555 484300
Jude 1230600
657555 379600
657555 23900
657555 1918700
Jude 45800
657555 994200
657555 478200
Jude 1022600
657555 760600
Jude 1667900
Jude 1140800
Jude 600200
657555 1059200
657555 966800
657555 1171000
Jude 379900
Jude 1315800
657555 1240800
Jude 1398000
Jude 664600
Jude 355300
657555 1433200
657555 349800
Jude 919300
Jude 1393100
657555 1077000
657555 893100
Jude 661000
Jude 1309600
Jude 1095100
657555 308100
657555 393100
657555 1093000
Jude 25400
657555 1536100
657555 1271000
Jude 1093700
Jude 1620400
657555 378600
Jude 882300
657555 1136900
657555 1371900
Jude 1740600
657555 1687700
657555 1847000
657555 1457200
657555 1298900
657555 13200
657555 643400
Jude 364800
657555 1643300
Jude 469900
657555 1600400
Jude 93500
657555 898400
657555 1019100
Jude 1261100
Jude 963900
Jude 1276500
657555 1772500
Jude 137600
Jude 1532200
Jude 1993100
Jude 429100
657555 159100
657555 341300
657555 1466000
Jude 1383500
657555 1385700
Jude 989800
657555 280200
Jude 1358600
Jude 1832700
657555 1812800
Jude 294700
657555 1728100
Jude 293300
Jude 647400
Jude 523600
657555 63500
657555 306200
657555 288000
657555 644800
Jude 970900
Jude 890500
657555 660000
Jude 1095900
657555 505500
Jude 210500
Jude 1717700
Jude 1859500
Jude 279400
Jude 806600
Jude 1611200
657555 393000
657555 355100
657555 401100
Jude 664200
657555 200200
Jude 1512400
Jude 1148600
Jude 815800
Jude 786300
Jude 290200
657555 130900
657555 225700
657555 1481300
Jude 1547000
Jude 1004500
657555 1854200
657555 328500
657555 1616600
657555 1536600
657555 132900
657555 1661900
657555 488800
Jude 1405200
657555 292500
Jude 561000
657555 513700
Jude 1664900
657555 1246600
657555 1841400
Jude 619800
657555 387400
Jude 1538900
657555 409500
657555 778100
Jude 1130800
Jude 380100
657555 11400
657555 1672800
Jude 1006900
657555 1282400
Jude 1036600
Jude 552800
657555 1965400
657555 1688300
Jude 938900
657555 1756600
Jude 596900
657555 333400
657555 679900
Jude 1318700
Jude 145400
Jude 922300
657555 354700
Jude 852900
Jude 990300
Jude 1705700
657555 1515200
657555 217800
657555 592100
Jude 915000
Jude 112500
Jude 1910600
657555 830000
Jude 584200
Jude 197100
657555 423800
657555 1405800
657555 1913600
Jude 1947900
657555 1192900
Jude 1921100
Jude 15200
Jude 1528200
Jude 86000
Jude 1094200
Jude 1904700
Jude 1071100
657555 1781400
Jude 1884500
Jude 404700
Jude 852300
Jude 383000
Jude 323500
657555 1265300
Jude 172500
657555 347400
Jude 1954900
657555 1437000
Jude 1258300
Jude 244400
657555 1119300
657555 693900
657555 517700
657555 1493900
Jude 737800
657555 1633000
657555 414900
Jude 1109100
Jude 1308200
657555 347000
657555 1392500
657555 1127700
Jude 851600
Jude 1772700
Jude 453700
657555 1384000
Jude 898200
Jude 685900
657555 561200
657555 457400
657555 47000
657555 1838700
657555 152000
657555 930200
657555 1880600
Jude 868400
Jude 263600
Jude 1011700
657555 1262300
Jude 208800
657555 1544400
Jude 1466200
Jude 1729100
Jude 1050100
657555 100600
657555 1937600
Jude 1653500
Jude 1831400
657555 1865300
Jude 371300
657555 806000
657555 150900
Jude 1702700
657555 798700
657555 1463400
657555 1719800
Jude 1854700
657555 1790200
657555 216100
657555 1345900
Jude 74300
Jude 1365000
Jude 571600
Jude 1666600
Jude 1214100
657555 1526700
Jude 873300
Jude 657400
657555 76200
Jude 1970100
657555 1373000
657555 186900
Jude 179200
657555 996800
Jude 877500
657555 1899200
Jude 1931300
Jude 404000
657555 1370000
657555 1597000
657555 966600
Jude 624600
Jude 1785300
657555 1976000
657555 1272100
Jude 1687500
Jude 696900
Jude 800800
657555 1274400
657555 55200
Jude 642600
657555 16900
657555 987200
Jude 1184700
Jude 1517400
657555 906100
Jude 1907300
Jude 939000
657555 1823400
Jude 416400
657555 1571200
657555 1615200
657555 1794400
Jude 1149500
657555 539800
Jude 1302400
657555 842600
Jude 1831900
657555 1208300
657555 232000
657555 1995900
657555 247000
Jude 843600
Jude 1678300
Jude 1113300
Jude 501200
Jude 1596500
Jude 341100
Jude 1702400
657555 1998600
Jude 1732600
657555 763400
657555 988200
Jude 610200
657555 940800
Jude 114500
Jude 433400
Jude 1438700
Jude 503200
657555 1370200
657555 653600
657555 1537400
657555 14700
657555 152100
Jude 1784600
Jude 347300
657555 831400
Jude 197600
Jude 1286300
657555 1410000
657555 1607000
Jude 1226800
657555 1611900
Jude 1660000
Jude 1860700
Jude 1785400
657555 221800
Jude 788800
Jude 871000
657555 1532600
Jude 517700
657555 1664100
Jude 364900
657555 1725300
657555 1458800
Jude 1114300
657555 1929600
657555 94100
Jude 590100
Jude 495000
Jude 1676700
Jude 1761600
657555 1180200
657555 1077400
Jude 1219300
Jude 9200
657555 434800
657555 1514000
Jude 1526600
Jude 1658200
657555 1763100
657555 10000
657555 1489400
657555 1305800
Jude 1888800
Jude 472000
657555 1348400
Jude 282100
Jude 880800
657555 1798600
657555 545700
Jude 1348300
657555 1799100
Jude 1573900
657555 624100
Jude 1300800
657555 1161200
Jude 511600
657555 440000
657555 1018200
Jude 241800
Jude 102900
657555 1503700
Jude 1813200
657555 301400
657555 1170700
Jude 701700
657555 1892200
657555 67500
Jude 254200
657555 76900
657555 1394100
Jude 1059000
Jude 1429800
657555 28200
657555 481100
657555 714800
Jude 1556000
657555 55800
Jude 261800
Jude 102500
657555 1320100
657555 290300
657555 1895400
Jude 26400
Jude 1736500
Jude 1206700
657555 1933000
657555 404000
657555 781900
Jude 1790600
657555 97900
Jude 465500
657555 508000
Jude 518700
657555 1992400
657555 282300
657555 107500
657555 1416700
Jude 66900
657555 1771500
Jude 1095900
657555 640600
Jude 364800
657555 1981200
657555 145800
Jude 166500
Jude 1106900
657555 1276100
Jude 565600
Jude 1662900
657555 1899700
Jude 1213800
Jude 822600
Jude 1975800
657555 587200
657555 1310600
657555 1346700
657555 17200
Jude 623100
657555 474700
Jude 1975400
Jude 1793100
Jude 1662200
657555 1584000
Jude 132400
Jude 1200300
657555 794100
657555 736600
Jude 1133700
Jude 182500
657555 1813800
Jude 231400
657555 2000000
Jude 1378900
Jude 1320400
Jude 60600
657555 1963300
657555 1439900
657555 531800
Jude 1867700
Jude 733300
657555 272000
657555 742400
Jude 1223300
Jude 980600
657555 388500
Jude 422700
657555 733800
657555 1303100
657555 1607700
657555 553600
Jude 1657700
Jude 956900
Jude 787000
Jude 853000
657555 1852300
Jude 1238400
Jude 1147900
657555 1704000
Jude 1867200
Jude 35600
Jude 70500
657555 100500
657555 740400
657555 469300
Jude 280800
Jude 367700
Jude 1235100
657555 392000
657555 526700
657555 1457000
Jude 1377700
Jude 716100
657555 147500
657555 1061100
657555 945600
657555 607300
Jude 562900
657555 1681500
Jude 1295900
657555 869600
Jude 972900
657555 309500
657555 162100
Jude 1273400
657555 1854500
657555 416600
Jude 1109500
657555 224100
657555 1249200
Jude 1346000
Jude 866300
657555 886400
657555 1849300
Jude 766900
Jude 132800
657555 1509200
657555 1949100
657555 882400
657555 629800
Jude 1846400
Jude 328000
657555 1013900
657555 646500
657555 934000
657555 1677100
Jude 25700
Jude 1779000
Jude 1228400
Jude 365900
Jude 999600
657555 1578700
657555 1921300
Jude 630500
657555 1514100
Jude 1735300
Jude 1346700
Jude 539800
657555 446300
657555 1982100
657555 79900
657555 1175900
Jude 1731800
Jude 939300
Jude 1563700
657555 836100
657555 151100
657555 825000
657555 1756900
657555 1321700
Jude 1497200
657555 1649000
657555 764700
657555 28000
Jude 1000600
657555 1575200
Jude 539700
657555 525700
657555 1222000
657555 1756800
Jude 1154000
657555 53400
657555 828600
Jude 198200
657555 1827000
657555 715900
657555 1046000
Jude 1182900
657555 659800
657555 1155000
Jude 1334600
Jude 846800
657555 1850300
657555 1670700
Jude 505700
657555 441900
Jude 225900
657555 1018000
Jude 1134200
657555 1591600
Jude 849100
Jude 1278000
657555 588300
657555 1220400
657555 1232200
657555 1905800
Jude 1201300
657555 956700
657555 453300
657555 1502100
657555 1350100
Jude 1318800
Jude 1740700
657555 1974200
657555 529600
657555 910000
Jude 457500
657555 1124300
Jude 1044800
657555 526100
657555 475100
657555 363700
Jude 140400
Jude 1099900
657555 1204000
Jude 407000
Jude 492900
Jude 1599600
657555 234400
Jude 1392300
657555 1362300
657555 1873400
Jude 1572300
Jude 1553200
657555 989500
Jude 996100
657555 1814100
657555 394000
657555 1501200
Jude 298100
657555 866000
657555 1919400
Jude 279200
Jude 1322500
Jude 1371600
657555 1623200
Jude 1217800
657555 1126500
Jude 230900
657555 1099900
657555 91400
657555 57600
Jude 657800
657555 1151500
657555 629300
Jude 1944100
657555 1614500
657555 1177600
Jude 228200
657555 1039800
Jude 1183000
657555 1747200
Jude 1731400
Jude 1191500
Jude 513800
657555 892400
657555 1520600
657555 229900
657555 899600
Jude 1757100
Jude 788500
Jude 1840800
Jude 1518600
657555 571800
Jude 1419800
657555 1876600
657555 843300
657555 361000
Jude 736300
Jude 1445200
657555 345400
Jude 1319700
Jude 567000
657555 1205700
657555 1728500
Jude 983900
Jude 711100
657555 553400
Jude 395400
Jude 1910900
Jude 1819100
Jude 392700
Jude 1324000
657555 107400
Jude 1372700
657555 1077700
Jude 916900
Jude 731500
657555 1197200
Jude 1799200
Jude 1355800
Jude 952000
657555 1858400
657555 1982300
Jude 936800
Jude 948600
657555 1182400
Jude 1444900
Jude 889300
657555 232700
Jude 303000
657555 474800
657555 81500
657555 724800
Jude 868500
657555 673200
Jude 1916800
657555 1400200
Jude 1634200
Jude 308700
Jude 1746700
657555 1693500
Jude 690600
657555 309000
Jude 1081000
657555 1636500
657555 432900
657555 1957600
657555 66700
Jude 1223400
657555 1016900
Jude 899700
657555 333100
Jude 842500
657555 85600
657555 999900
657555 544600
Jude 1065800
Jude 1081700
657555 445300
Jude 395400
657555 1797300
657555 204300
657555 1466000
Jude 894400
Jude 728200
657555 307800
657555 1589500
Jude 1456700
Jude 20900
657555 624500
Jude 914600
Jude 139400
Jude 747900
657555 671300
657555 222300
Jude 1614000
Jude 94900
657555 1313500
Jude 182700
Jude 1671300
Jude 1284600
657555 1022700
657555 1264900
657555 1857100
Jude 430100
Jude 78000
657555 673200
Jude 998300
Jude 1109800
657555 1615100
657555 1852000
Jude 518200
657555 1068800
657555 1892300
Jude 531600
Jude 368200
Jude 290700
657555 382500
Jude 692600
657555 1112700
657555 78200
Jude 157500
657555 725900
657555 343500
Jude 452400
Jude 1275200
657555 570600
Jude 1522600
Jude 825700
657555 743800
657555 86900
Jude 97000
657555 689700
Jude 1839700
Jude 1106800
657555 1298500
657555 1429600
657555 871200
Jude 321100
Jude 1783300
657555 345300
Jude 1450700
Jude 446900
Jude 1551300
657555 751000
657555 474600
657555 741200
Jude 1351300
Jude 1761100
Jude 130500
Jude 306000
Jude 781800
Jude 824800
657555 1224500
657555 1516400
657555 1536200
657555 1017800
657555 1217500
657555 543000
657555 1334200
Jude 861100
657555 904300
Jude 1603800
657555 1287000
657555 172400
657555 724600
657555 940200
657555 1528600
Jude 758000
657555 1866100
657555 94600
657555 1068900
Jude 420700
Jude 623900
657555 1453100
Jude 1906800
657555 724800
657555 1870600
657555 929900
Jude 396200
657555 755200
Jude 1238000
657555 1082100
Jude 764600
657555 943100
Jude 336600
Jude 1150700
657555 274900
Jude 170100
657555 1519300
657555 798800
657555 1450100
657555 1914400
Jude 622200
Jude 505600
657555 520400
Jude 315500
Jude 1298100
657555 568700
657555 1391400
Jude 1191500
657555 1826400
Jude 381300
657555 1559300
657555 973500
Jude 687900
657555 1586500
657555 97500
657555 1183200
Jude 1728100
657555 1588300
Jude 515900
Jude 1734000
657555 1685000
Jude 1868900
657555 19400
Jude 1349800
Jude 107500
Jude 639200
Jude 223500
Jude 1009300
657555 1112800
Jude 889700
Jude 1230900
Jude 1611800
Jude 1945600
Jude 1792300
Jude 851500
657555 1495500
657555 833400
Jude 803100
657555 1777200
657555 636900
657555 210000
Jude 1433800
657555 1745000
Jude 725700
657555 1173900
Jude 282200
Jude 1367300
657555 860300
657555 1136500
657555 1330400
657555 1552300
Jude 617700
657555 10700
Jude 160000
657555 973600
657555 227000
657555 16200
Jude 463600
Jude 405200
657555 1634900
Jude 1178600
