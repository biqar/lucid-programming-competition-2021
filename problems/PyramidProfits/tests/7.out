657555 304546260
Jude 683648640
